Assets/Scripts/Game System/LevelEndTriggerManager.cs

```diff
@@ -1,7 +1,7 @@
 using UnityEngine;
 using UnityEngine.SceneManagement;
 
-public class LevelEndTriggerController : MonoBehaviour
+public class LevelEndTriggerManager : MonoBehaviour
```

Comment:
I think for class naming:
- Controller should be used for scripts that control the behavior of a single entity.
- Manager should be used for scripts that control multiple entities.
- Handler should be used for a class that mainly contains an event handler method or methods.

By entity, I mean an individual thing in the game that can consist of multiple GameObjects.